### ISSUE TITLE
Incorrect Count of projects completed

### ISSUE THREAD

**Issue by tarik-ali** [freeCodeCamp/freeCodeCamp#34396]
**Describe the bug**
The number of projects I completed and the number of projects displayed on the home page as completed do not match.

**Screenshots**
![capture](https://user-images.githubusercontent.com/44884329/48790271-d5777400-eca3-11e8-8568-f12919216432.PNG)
![capture2](https://user-images.githubusercontent.com/44884329/48790281-de684580-eca3-11e8-8b3b-d8324e73c884.PNG)

**Desktop (please complete the following information):**
User Agent is: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.102 Safari/537.36.

**Comment by richb999git** [comment]
I had the same issue. I have completed all the projects and have 6 certificates but only 25 out of 30 projects completed according to my home page. It has something to do with the Javascript Algorithms and Data Structures Projects not being counted towards the completed project count. They are completed in a different way from the other projects. I have looked through the code and I think it is something to do with "completedProjectCount" and "challengeTypes" (3 or 4) but I can't work out how Javascript Algorithms and Data Structures Projects are treated differently. Maybe need to find where challenges (projects) are marked as completed.
